creator-node/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "baseUrl": "./src",
     "pretty": true,
     "target": "es2020",
-    "module": "es2020",
+    "module": "commonjs",
```

Comment:
this is the actual fix, confirmed by dylan with link to official tsconfig files https://github.com/tsconfig/bases/blob/main/bases/node16-strictest.combined.json#L10